src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = days * 40;
+
+  if (days >= 7) { // discount is $50 for 7 or more days
+    cost -= 50;
+  } else if (days >= 3) { // discount is $20 for 3 or more days
+    cost -= 20;
+  }
```

Comment:
fine, but could be even simpler
```
if (days >= 7) {
  return cost - 50
}

if (days >= 3) {
  return cost - 20
}
```

and please make 20 and 50 meaningful - i.e. assign to variables with explanatory names